app/controllers/concerns/foreman/controller/parameters/nic_base.rb

```diff
@@ -24,6 +23,7 @@ def add_nic_base_params_filter(filter)
         :username,
         :virtual,
         {:attached_devices => []},
+        {:compute_attributes => [:bridge, :from_profile, :model, :network, :type, :name]},
```

Comment:
@domcleal:

> compute_attributes is generally passed through without filtering

I saw that, but thought the whole point of strong parameters was filtering.





> That keep_param helper only works for the first level of hashes at the moment though, so it might need extending somehow to work in Host for nested interfaces + compute_attributes.

Do you want to suggest something? I'd be happy to look at the PR and test it with vmware.